iOSNativeSwiftTemplate/iOSNativeSwiftTemplate/AppDelegate.swift

```diff
@@ -34,6 +34,12 @@ class AppDelegate : UIResponder, UIApplicationDelegate {
     override init() {
         super.init()
         MobileSyncSDKManager.initializeSDK()
+        
+        // Uncomment when disabling log in via Salesforce UI Bridge API generated QR codes
```

Comment:
Could this be moved to a helper method like `registerForRemotePushNotifications`? The helper call can still be commented out but compiling can catch any Mobile SDK API changes in the future. 

Also for "Uncomment when disabling log in", it looks like it's for enabling? Or am I misunderstanding?